docker_test.go

```diff
@@ -1267,3 +1268,31 @@ func TestContainerNonExistentImage(t *testing.T) {
 	})
 
 }
+
+func TestDockerContainerCopyFileToContainer(t *testing.T) {
```

Comment:
I added a single test using CopyFileToContainer since it uses already CopyToContainer underneath